components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -6174,6 +6174,35 @@ public String addBlockCondition(String conditionType, String conditionValue) thr
         return createdBlockConditionsDto.getUUID();
     }
 
+    /**
+     * Add Block Condition with condition status
+     *
+     * @param conditionType type of the condition (IP, Context .. )
+     * @param conditionValue value of the condition
+     * @param conditionStatus status of the condition
+     */
```

Comment:
Shall we add the override annotation here since this method overrides the method declared in the APIProvider.java interface?
Shall we also remove the java docs comments since the comments are already available in the method declared in the APIProvider.java interface?